src/components/Embeds/AirMap.js

```diff
@@ -4,9 +4,11 @@ import React from "react";
 import { CITIES_LOCATION } from "api";
 
 function AirMap({ city }) {
+  const appMapUrl = process.env.NEXT_PUBLIC_APP_MAP_URL;
```

Comment:
Pick one approach and stick with it e.g. `const appMapUrl = process.env....` or `const APP_MAP_URL = process.env....`